### PR TITLE
Invalid Compilers for Perlmutter in Buildtest Configuration File - Fix

### PR DESCRIPTION
**Pull request by braeeast2001**

### CODE REVIEW

**Comment by braeeast2001** [comment]
@shahzebsiddiqui  When I run "buildtest config compilers test" no errors appear. Please see below
Screen Shot 2023-07-31 at 4 45 39 PM

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (shahzebsiddiqui) [inline .idea/.gitignore:3]: please remove this file

File: .idea/buildtest-nersc.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (shahzebsiddiqui) [inline .idea/buildtest-nersc.iml:8]: please remove this file

**Comment by shahzebsiddiqui** [comment]
> @shahzebsiddiqui When I run "buildtest config compilers test" no errors appear. Please see below Screen Shot 2023-07-31 at 4 45 39 PM

You need to run `buildtest config compilers find --update` and this will update the configuration file. You can then run `buildtest cg co test`

**Comment by shahzebsiddiqui** [comment]
@braeeast2001 thanks for removing all the files. Please add the config.yml file with the updated compilers.

**Comment by shahzebsiddiqui** [comment]
@braeeast2001 can you please open a new MR. There is something wrong with commit history i forced push to the `devel` branch of this repo

**Pull request closed without merging**